daemon/daemon.go

```diff
@@ -612,10 +612,9 @@ func (d *daemon) RestartTeamLab(req *pb.RestartTeamLabRequest, resp pb.Daemon_Re
 		return err
 	}
 
-	lab, err := ev.GetHub().GetLabByTag(req.LabTag)
-
-	if err != nil {
-		return err
+ 	lab,ok:=ev.GetLabByTeam(req.TeamId);
```

Comment:
Again, `go fmt` or `goimports` needs to be added to your IDE, so it formats the code when saving it.

Reply:
enabled filewatcher on golang. Thank you for the feedback :)